Update credit.md to add alt text and break the chapter into smaller subchapter (#1922)
alan-turing-institute/the-turing-way: book/website/communication/citable.md

| {ref}`Overview of Reproducibility<rr-overview>` | Useful but not essential |
| {ref}`Open research<rr-open)>` | Useful but not essential |

## Summary

Many researchers may argue that spending time on applying reproducible research practices will reduce the time they have available for activities by which they received credits that are traditionally measured, such as writing papers.

What if you could get credit for your reproducibility efforts as well?
In this chapter, we will describe how working reproducibly will allow you to get most of your research outputs published in a way that can be cited by others (not just your paper)!

```{figure} ../figures/OrcidIDs.jpg
---
name: OrcidIDs
alt: This image shows a female researcher in the centre with her arms raised. She is surrounded in an orbit by different types of documents. On the top left, there is a caption stating 'ORCID IDS', while on the bottom right there is the text that says - 'One ring to rule all outputs'.
---
_The Turing Way_ project illustration by Scriberia.
Used under a CC-BY 4.0 licence.
DOI: [10.5281/zenodo.3332807](https://doi.org/10.5281/zenodo.3332807)
```

## Background & Motivation

> Academic research is a reputation economy, and citations are the currency.

Author: That would be nice. Would you like to suggest a rephrasing?

> Academic research has traditionally had a reputation economy, and citations are considered a major metrics of success in academia.

Author: ```suggestion
Academic research has traditionally maintained a 'reputation economy' where citations are the currency or considered a major metric of success.
```

This is a well-established practice, and while it has its structural (academic) problems at least all stakeholders understand what's involved.
One of the consequences of this system is that labour which *doesn't* result in published articles tends to be ignored, discouraging researchers from making their data more open or specialising in software development.

Establishing good citation practices using persistent and citable identifiers for non-article content is a step towards recognising this valuable work and encouraging more people to take it up.
If you can demonstrate the impact of your reproducible research work in addition to more traditional research outputs, you can justify spending more time on doing things right.

We describe, how you can {ref}`prepare your research object to be citable<rr-credit-make>`, {ref}`how others can cite them<rr-credit-cite>` (or you can cite others work) and how you can use them to {ref}`gain recognition and build your profile<rr-credit-recognise>`.

(rr-credit-contribute)=
## Contribute to this Chapter

> We invite you to contribute to this chapter by completing the following aspects:

Author: ```suggestion
We invite you to contribute to this chapter by writing about one or multiple of the following aspects:
```

- Demonstrate impact with various metrics described in this chapter

> - Add examples for DOIs from data journals

Author: ```suggestion
- Add some tips for peer-reviewed publishing of research components with DOIs (like data journals, JOSS for software)
```
